Todo.py

```diff
@@ -0,0 +1,203 @@
+import discord
+from settings import *
+from checks import *
+
+TOKEN = "NTY4MDkyNTY5NTE5NzgzOTM2.XMiBDw.zCL0KhV_gOtxfyACctgYCLcLlEY"
+# PREFIX = "g:"
+
+# EMOJIS = {
+#     "wastebasket": "\U0001F5D1",
+#     "check": "\U00002705",
+#     "hourglass": "\U000023F3"
+# }
+
+#bot = commands.Bot(command_prefix=PREFIX)
+
+class Todo(commands.Cog):
+    """A suite of command to make a nice todo list."""
+    def __init__(self, bot):
+        self.bot = bot
+
+    @commands.Cog.listener()
+    async def on_message(self, message):
+        if message.author == bot.user:
+            return
+        await bot.process_commands(message)
+
+    @commands.Cog.listener()
+    async def on_raw_reaction_add(self, reaction):
+        if reaction.user_id != bot.user.id:
+            message = await bot.get_channel(reaction.channel_id).fetch_message(reaction.message_id)
+
+            with open(TODO_CHANNEL, "r") as file:
```

Comment:
is it a channel or a file ? This seems unclear to me...

Reply:
Also where do you define this variable ?

Reply:
It's an error from, I corrected that, It's a file I also fixed that

Reply:
Ok I'll wait for the commit then ;)